### PR TITLE
Add lockdownStarted and lockdownEnded hooks

### PR DESCRIPTION
**Pull request by WilliamVenner**

### CODE REVIEW

**Comment by mark27205** [comment]
Yeah, would appreciate that.

File: gamemode/modules/police/sh_interface.lua [hunk 1]
```diff
@@ -185,3 +185,23 @@ DarkRP.hookStub{
         }
     }
 }
+
+DarkRP.hookStub{
```
Comment (mark27205) [inline gamemode/modules/police/sh_interface.lua]: Maybe add a space between the b and the { to follow the coding style.

File: gamemode/modules/police/sh_interface.lua [hunk 2]
```diff
@@ -185,3 +185,23 @@ DarkRP.hookStub{
...
+    returns = {
+        {
+            name = "actor",
+            description = "The player who started the lockdown",
+            type = "Player"
+        }
+    }
+}
+
+DarkRP.hookStub{
```
Comment (mark27205) [inline gamemode/modules/police/sh_interface.lua]: Maybe add a space between the b and the { to follow the coding style.

**Comment by FPtje** [comment]
Alright, thanks, but the hookstubs are off. You're giving both hooks a `ply` as parameter, but in the started hook it's said to be a return value, and in the end hook it's not specified at all. Let me fix it real quick.

**Comment by WilliamVenner** [comment]
My apologies... not sure how that managed to fly past my head :o

**Pull request merged**